### ISSUE TITLE
Is EMQ a good fit for my use case?

### ISSUE THREAD

**Issue by gihad** [emqx/emqx#1509]
Hi,

My use case is a mobile game that will have between 1 and 2 million online users simultaneously, and the server will send a question (a message) to all users at the exact same time and within 10 seconds all users will reply with an answer choice to the server. 
Since this is a live game show, all of this is time sensitive, we expect all 1 million+ users to get the question at around the same time and when they all reply we need to calculate the results within a few seconds. This pattern of sending a question to all users and getting answers form all users repeats itself about once a minute.

Another game that does something similar uses Json over websockets, I think this can be improved by using the lightest possible protocol so I'm looking at MQTT,

Basically I'm motivated  to get messages form the 1million+ client into the server (and vice versa) as fast and as reliably as possible.

Is EMQ (and MQTT) a good fit for this?

Regarding this: "The 1.0 release of the EMQ broker has scaled to 1.3 million concurrent MQTT connections on a 12 Core, 32G CentOS server."  

I assume this doesn't mean 1.3 million messages a second, would I ever be able to achieve that performance?
Can I assume this will scale horizontally by adding more servers?

Thanks.

**Comment by gihad** [comment]
Here is a TL;DR version of the question:

I want to push a message to 1 million mobile clients and all the messages need to make it through in 1 second.  After this all the clients will reply back with a message to the server. We will need SSL.

Are these numbers feasible?

**Comment by gihad** [comment]
Is there another place I could ask these kind of questions? Slack group? IRC? I was able to ask this question in other communities such as VerneMQ.

**Comment by Gilbert-Wong** [comment]
Hi, @gihad , sorry for late respond, the slack channel can be joined now, [#emqx-users](https://emqx.slack.com/messages/CBUF2TTB8/), [#emqx-devs](https://emqx.slack.com/messages/CBSL57DUH/)
